src/koaApi.coffee

```diff
@@ -77,10 +79,27 @@ exports.setupApp = (done) ->
   app.use route.get '/visualizer/events/:receivedTime', visualizer.getLatestEvents
   app.use route.get '/visualizer/sync', visualizer.sync
 
+# -- Old metrics Routes for Backward Compatibility
   app.use route.get '/metrics', metrics.getGlobalLoadTimeMetrics
   app.use route.get '/metrics/status', metrics.getGlobalStatusMetrics
   app.use route.get '/metrics/:type/:channelId', metrics.getChannelMetrics
-  
+# -- End of Old Metrics Routes --
+
+# -- New metrics Routes --
+  app.use route.get '/stats', if statsd_instance.enabled then statsd.retrieveTransactionCountPerHour else metrics.getGlobalLoadTimeMetrics
+  app.use route.get '/stats/status', if statsd_instance.enabled then statsd.fetcGlobalStatusMetrics else metrics.getGlobalStatusMetrics
+  app.use route.get '/stats/:type/:channelId', if statsd_instance.enabled then statsd.retrieveChannelMetrics else metrics.getChannelMetrics
+  app.use route.get '/stats/load-time', if statsd_instance.enabled then statsd.retrieveAverageLoadTimePerHour else metrics.getGlobalLoadTimeMetrics
```

Comment:
Yes this is correct I separated the functionality in the statsd api the  metrics.getGlobalLoadTimeMetrics returns both averageLoadTime  and load in one call